testing/shadowsocks-libev/APKBUILD

```diff
@@ -1,38 +1,41 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=shadowsocks-libev
-pkgver=2.5.5
-pkgrel=3
+pkgver=3.2.5
+pkgrel=0
 pkgdesc="lightweight secured SOCKS5 proxy"
 url="https://shadowsocks.org/en/download/servers.html"
 arch="all"
-license="GPL-3.0"
-depends=""
-makedepends="asciidoc autoconf libtool linux-headers
-	libressl-dev pcre-dev xmlto libev-dev udns-dev libsodium-dev zlib-dev"
-install=""
-subpackages="$pkgname-dev $pkgname-doc"
+license="GPL-3.0-or-later"
+makedepends="asciidoc autoconf automake libtool linux-headers libcorkipset-dev
+	mbedtls-dev pcre-dev xmlto libev-dev c-ares-dev libsodium-dev zlib-dev libbloom-dev"
+subpackages="$pkgname-dev $pkgname-doc lib$pkgname:libs"
 source="shadowsocks-libev-$pkgver.tar.gz::https://github.com/shadowsocks/shadowsocks-libev/archive/v$pkgver.tar.gz"
-builddir="$srcdir/shadowsocks-libev-$pkgver"
+
+prepare() {
+	default_prepare
+	sed -i 's|AC_CONFIG_FILES(\[libbloom/Makefile libcork/Makefile libipset/Makefile\])||' configure.ac
```

Comment:
I believe we prefer patches rather than inline sed statements for these kinds of things.